sdk/eventhub/azure-eventhubs/HISTORY.md

```diff
@@ -1,10 +1,21 @@
 # Release History
 
-## 5.0.0b4 (2019-XX-XX)
+## 5.0.0b4 (2019-10-08)
 
 **New features**
 
-- Support for tracing  #7153
+- Added support for tracing (issue #7153).
+- Added the capability of `EventHubConsumer` to track last enqueued event properties of the partition.
+    - Added new boolean type parameter`track_last_enqueued_event_properties` in method `EventHubClient.create_consumer()`.
+    - Added new property `last_enqueued_event_properties` of on `EventHubConsumer` which contains sequence_number, offset, enqueued_time and retrieval_time information.
+    - By default the capability is disabled as it will cost extra band width for transferring more information if turned on.
+
+**Breaking changes**
+
+- Removed support for IoThHub direct connection.
+    - [EventHubs compatible connection string](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-read-builtin) of an IotHub can be used to create EventHubClient.
```

Comment:
It would be better to change "to create EventHubClient" to "to receive device-to-cloud IotHub messages". We also removed sending IoT cloud-to-device messages from Eventhub SDK. We better mention this and tell users to use IoT hub SDK instead. As far as I know no users use this function feature though.

Reply:
good catch, updated